odkx-src/docs-developer-guide.rst

```diff
@@ -341,6 +341,18 @@ specify the check name in the check list
 in the function :py:func:`exclude_checks`
 in the file :file:`style-test.py`.
 
+Excluding Spelling checks
```

Comment:
Capitalization is uneven here. Also a bit confusing "Excluding spelling checks." Maybe instead "Excluding specific words from the spelling check" or something similar. You aren't canceling the whole check.